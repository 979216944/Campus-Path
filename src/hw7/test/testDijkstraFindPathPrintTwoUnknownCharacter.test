# Test Dijkstra finding a path,
# it prints two lines complaining,
# "unknown character notFlower"
# "unknown character notFlower"
# if notFlower1 and notFlower2 are both not in graph.

# load graph
LoadGraph g flowers.tsv

# find a minimum-cost path with both origin and dest node not in the graph
FindPath g notFlower1 notFlower2