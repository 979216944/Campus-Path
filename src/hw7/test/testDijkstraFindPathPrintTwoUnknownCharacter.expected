# Test Dijkstra finding a path,
# it prints two lines complaining,
# "unknown character notFlower"
# "unknown character notFlower"
# if notFlower1 and notFlower2 are both not in graph.

# load graph
loaded graph g

# find a minimum-cost path with both origin and dest node not in the graph
unknown character notFlower1
unknown character notFlower2